library/src/androidMain/kotlin/com/lightningkite/kiteui/views/direct/TextField.android.kt

```diff
@@ -181,10 +187,6 @@ var EditText.keyboardHints: KeyboardHints
 
             KeyboardType.Phone -> InputType.TYPE_CLASS_PHONE
             KeyboardType.Email -> InputType.TYPE_CLASS_TEXT or InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS
-        }.let {
```

Comment:
Can we keep this?

Reply:
Autofill behavior for passwords is still set on line 196. The removed block of code was only setting the masking dots, which we now control by manipulating the `EditText.transformationMethod`